ovos-i2csound

```diff
@@ -38,8 +38,52 @@ detect_i2c_device() {
     fi
 }
 
+pulseaudio_config() {
+    echo "configuring pulseaudio"
+    # Set default configurations
+    if [ -f /var/lib/alsa/default-asound.state ]; then
+        ASOUND_STATE=/var/lib/alsa/default-asound.state
+    fi
+    if [ -f /etc/pulse/system.pa.d/pulseaudio-system.pa ]; then
+        PULSE_SYSTEM=/etc/pulse/system.pa.d/pulseaudio-system.pa
+    fi
+    if [ -f /etc/pulse/daemon.conf.d/pulseaudio-daemon.conf ]; then
+        PULSE_DAEMON=/etc/pulse/daemon.conf.d/pulseaudio-daemon.conf
+    fi
+
+    # Remove old configurations
+    if [ -f /etc/pulse/system.pa ] || [ -L /etc/pulse/system.pa ]; then
+        rm /etc/pulse/system.pa
+    fi
+    if [ -f /etc/pulse/default.pa ] || [ -L /etc/pulse/system.pa ]; then
+        rm /etc/pulse/default.pa
+    fi
+    if [ -f /etc/pulse/daemon.conf ] || [ -L /etc/pulse/daemon.conf ]; then
+        rm /etc/pulse/daemon.conf
+    fi
+    if [ -f /var/lib/alsa/asound.state ] || [ -L /var/lib/alsa/asound.state ]; then
+        rm /var/lib/alsa/asound.state
+    fi
+
+    if [[ $1 ]] && [[ -f $1 ]]; then
+        PULSE_SYSTEM=$1
+    fi
+    if [[ $2 ]] && [[ -f $2 ]]; then
+        PULSE_DAEMON=$2
+    fi
+    echo "Using .pa file $PULSE_SYSTEM"
+    echo "Using daemon config file $PULSE_DAEMON"
+    echo "Using asound.state file $ASOUND_STATE"
+    ln -s $PULSE_SYSTEM /etc/pulse/system.pa
```

Comment:
With drop-in files this is also no longer needed.

Just leave all the default .pa and .conf files alone and the drop-in support of pulseaudio will load the additional rules/configurations set within them.

Reply:
What happens when defaults need set in the `.pa` files? such as [here](https://github.com/OpenVoiceOS/ovos-i2csound/blob/df35ec47cb7c69617c3e6b0d6e552b322dca2fbe/pulseaudio/sj201-default.pa#L69).  Wouldn't that override any other source because the other files do not have that?

Reply:
As long as it is valid .pa and/ or .conf configuration, they are applied.